components/brave_ads/common/features.cc

```diff
@@ -152,5 +155,9 @@ int AdNotificationInsetY() {
 
 #endif  // !defined(OS_ANDROID)
 
+bool IsRequestAdsEnabledApiEnabled() {
```

Comment:
there's really no need to wrap this in a method if it only checks the feature

Reply:
Agreed, but I have followed the style how code was written in this header, and I think it is better to follow the same style as code around. WDYT?

Reply:
I guess both patterns are used (direct feature check and wrapper method. Not sure which is preferred or why